test/Interop/SwiftToCxx/stdlib/swift-synchronization-in-cxx.swift

```diff
@@ -0,0 +1,17 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend -typecheck %s -typecheck -module-name UseSynchronization -enable-experimental-cxx-interop -clang-header-expose-decls=all-public -emit-clang-header-path %t/UseSynchronization.h
+// RUN: %FileCheck %s < %t/UseSynchronization.h
+
+// REQUIRES: objc_interop
```

Comment:
Do we rely on Obj-C runtime in this test? Should this be `// REQUIRES: synchronization` instead?